meson.build

```diff
@@ -1575,105 +1578,135 @@ exes += exe
 manadocs += ['sys-utils/ctrlaltdel.8.adoc']
 bashcompletions += ['ctrlaltdel']
 
+opt = get_option('build-fsfreeze').require(cc.has_header('linux/fs.h')).allowed()
```

Comment:
Why not fetch this from `conf` where it has already been checked?

Reply:
Oops, my bad. I'll do that.

Reply:
Done.